doc/release-process.md

```diff
@@ -64,14 +60,14 @@ This will perform a few last-minute consistency checks in the build system files
 
 ### First time / New builders
 
-If you're using the automated script (found in [contrib/gitian-build.py](/contrib/gitian-build.py)), then at this point you should run it with the "--setup" command. Otherwise ignore this.
+Install Guix using one of the installation methods detailed in
+[contrib/guix/README.md](contrib/guix/README.md).
```

Comment:
In 5016091dd8b827c74fe28212be43d3b22014c5b6 "guix: Adapt release-process.md to new Guix process"

I think this should refer to `contrib/guix/INSTALL.md`

Reply:
Fixed, thanks!